components/common/src/command/package/install.rs

```diff
@@ -57,6 +57,7 @@ pub const RETRY_WAIT: u64 = 3000;
 
 pub fn start<P1: ?Sized, P2: ?Sized>(ui: &mut UI,
                                      url: &str,
+                                     channel: Option<&str>,
```

Comment:
@fnichol I think we can probably hit that in a later pass. I think we're going to have one more pass on channels even though this pushes us over the finish line for now